rpa-builder/6.2/modules/ROOT/pages/toolbox-excel-operations-excel-controlled-loop.adoc

```diff
@@ -16,30 +16,56 @@ session. You enter the source file in the Excel Session.
 
 *Excel Operation Settings*
 
-* *Column* The Excel sheet is read from this column.
-* *Empty rows count for break* Specifies the number of consecutive empty
+* *Column* 
++
+The Excel sheet is read from this column.
+* *Empty rows count for break* 
++
+Specifies the number of consecutive empty
 rows that the loop is to be terminated after.

```

Comment:
I know this isn't green, but I found this confusing. Does the loop get terminated after the specified number of empty rows is reached? 
Maybe:
Terminates the loop after the specified number of consecutive rows is reached.